source/gui/__init__.py

```diff
@@ -31,6 +31,7 @@
 	# be cautious when removing
```

Comment:
general notes:

- there's a lot of strings that are not translatable which should be
- there's a lot of `snake_case` not `camelCase`
- can you ensure all usages of `isModalMessageBoxActive` get updated to focus the dialogs where appropriate

Reply:
I believe these have all been addressed.